recipe-box-backend/src/main/java/com/endava/recipebox/controller/RecipeController.java

```diff
@@ -0,0 +1,51 @@
+package com.endava.recipebox.controller;
+
+import com.endava.recipebox.model.MealType;
+import com.endava.recipebox.response.CustomRecipeMapper;
+import com.endava.recipebox.service.RecipeService;
+import org.mapstruct.factory.Mappers;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+
+
+@RestController
+@RequestMapping(path = "/recipe_box", produces = "application/json")
```

Comment:
receipe_box si the name of the entire application, in this case we need to reflect the controller, so I recommand to use only "/recipes" inside RequestMapping.